TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -50,6 +50,8 @@ def __init__(self, *args, **kwargs):
     """
     AgentModule.__init__(self, *args, **kwargs)
 
+    self.shifterProxy = None
```

Comment:
All this shifterProxy business is already handled in the base class. Why is it needed here?

Reply:
This may be needed for running this agent for multi-VO installations. This was done by @andresailer who may comment.

Reply:
Right, I did change this... 
So it used to be am_setOption (https://github.com/DIRACGrid/DIRAC/commit/941ed9f037b9ddea22647f782ca6c4e61b6a3cd1) for the shifterProxy, which was changed to getOption to run without a shifterProxy.

Later the existence of a shiterproxy value is later used to determine if to use the shifterproxy or the owner of the transformation.

Thus, it is cleaner to define this option here in the init and not just in initialize, (unless the magic in the AgentModule somehow determines that the existence of the shifterProxy attribute even if None means there is a shifterProxy and tries to set it up...)

Reply:
I believe the AgentModule sets the shifter proxy as soon as the option is set in the CS. But it is not quite what you want I understand ? 
Maybe an inline comment about this would help making sure we do not remove it in the future, believing that AgentModule does the job.

Reply:
The shifterProxy member is not set for the agent at all, is it? It is only about the value in the configuration?
(Should rename the member....)
If the shifterProxy member were to be removed, these kind of lines would crash

https://github.com/DIRACGrid/DIRAC/blob/6c94a8da5125cfd653dbb3a786ddcac4df870b9b/TransformationSystem/Agent/TransformationCleaningAgent.py#L143-L146